src/geo/ui/widgets/histogram/model.js

```diff
@@ -58,12 +58,26 @@ module.exports = WidgetModel.extend({
   },
 
   parse: function(data) {
+    var numberOfBins = data.bins_count;
+    var width = data.bin_width;
+    var nulls = data.nulls_count;
+    var start = data.bins_start;
 
-    this._data.reset(data.bins);
+    var buckets = new Array(numberOfBins);
+    _.each(data.bins, function(b) {
+      buckets[b.bin] = _.omit(b, 'bins_number', 'bin_width', 'nulls_count');
```

Comment:
You don't need to do the omit here. It was like that server side due data being retrieved from database, however server side is already *cleaning* that for you :smile:.